deepchem/feat/graph_data.py

```diff
@@ -293,3 +336,12 @@ def __init__(self, graph_list: Sequence[GraphData]):
             edge_features=batch_edge_features,
             node_pos_features=batch_node_pos_features,
         )
+
+    def numpy_to_torch(self):
```

Comment:
Can you add a docstring here explaining the difference from the superclass implementation?